docs/inbound-transfers/ISA-Transfers.md

```diff
@@ -8,5 +8,8 @@ tags: [Inbound Transfers]
 Each inbound ISA transfer request must satisfy the following requirements:
 - The account that the portfolio is under must be an ISA.
 
+## ISA transfer history details
+After the funds were received and are available in the WealthKernel portfolio the status of the transfer will be updated to Transferred. To complete the transfer you be required to upload transfer history detalis, after the status will be updated to Completed.
```

Comment:
I would reword this as the tenant doesn't upload the transfer history form, and we also talk about it below too, maybe remove this section and rewrite the section below to say `After the inbound ISA transfer funds have been received... At this point the inbound transfer will be completed.`